CMSgov/cms-carts-seds
# MDCT-282 - Generate Form Templates

# Description
* BO Users Can generate form templates
* Fix for names of roles from IDM.
* Small correction for the state endpoint to allow sections 2 & 5 to load, placeholder for future work.
* This should hopefully not conflict with any of the other BO/Help/Approver tickets from this sprint, but a lot of the roles overlap. If anything looks off, let me know and I will correct it ASAP

## How to test

Seed data not setup in dev envs, must be run locally.
- Point your env file to the cognito pool for this branch. The branch has updated user roles. (Save your original file so you can roll back). If you prefer to get the info without digging around in aws, just message me and I can get you it.
- Make sure your .env has any new table names added from the example .env file. There are a couple wrapped in.
- BO Users should see a generate templates option on their landing page. This will generate templates for all states that do not already have a form generated for the given year.

## Dependencies

Correct roles in the cognito env matching the BO user.

# Get it done

I'd like to move this on through fast to get all the roles matching in each env

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
